scripts/release/__tests__/helpers.js

```diff
@@ -0,0 +1,28 @@
+import merge from "lodash.merge";
+
+/**
+ * @typedef {{ execa: { stdout: string; stderr: string }}} InjectedValues
+ */
+
+/**
+ * @returns {InjectedValues}
+ */
+export function getInjectedValues() {
+  return globalThis.INJECTED_VALUES;
+}
+
+export function updateInjectedValues(newValues) {
+  const currentValues = { ...globalThis.INJECTED_VALUES };
+  globalThis.INJECTED_VALUES = merge(currentValues, newValues);
```

Comment:
Using `globalThis` to store `INJECTED_VALUES` won't cause problems? The tests are async.